### DISCUSSION TITLE
Is there a way to clone project settings to a new project?

### DISCUSSION THREAD

**Discussion by Jaimy128** [community/community#44715] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I made a new project in github with custom fields etc,

is there a way to make like a template from a project, so that I can use it for every project? or do I just have to make these custom fields every time?

**Comment by SilasRodrigues19** [comment]
Hey @Jaimy128 

You can go to the repo `Settings`, and check the `Template repository` option 

![image](https://user-images.githubusercontent.com/47532703/213289123-6baefa7b-f4b5-4e70-827f-cc647103dd55.png)

**Reply by Jaimy128** [reply]
That doesn't take the linked project with it.
